rpa-manager/modules/ROOT/pages/organizationmanagement-overview.adoc

```diff
@@ -1,3 +1,4 @@
 = Organization Management Overview
+:page-notice-banner-message: MuleSoft RPA is integrating with Anypoint Platform to unify the login experience and provide support for external identity providers (IdP). After your organization is migrated to Anypoint Platform, the Organization Management module is replaced by the Settings module.
```

Comment:
```suggestion
:page-notice-banner-message: MuleSoft RPA is integrating with Anypoint Platform to unify the login experience and provide support for external identity providers (IdPs). After your organization migrates to Anypoint Platform, the Settings module replaces the Organization Management module.
```

Reply:
I am confused by the banner. It is integrating with Anypoint Platform but is not integrate yet? If it is not integrated yet why do you need the banner?

Reply:
Global for this banner